### ISSUE TITLE
Invalid Transaction when using Bank to make payment

### ISSUE THREAD

**Issue by Leekogram** [wilburt/flutter_paystack#158]
![Simulator Screen Shot - iPhone 14 Pro Max - 2023-04-18 at 09 41 53](https://user-images.githubusercontent.com/33817077/232727388-f6581cf7-b647-4331-a032-9b27d131288f.png)

Please all i get is Invalid Transaction when trying to make payment using bank below is my function. what could be the problem ?

  processPaystackPayment({required skKey, context}) async {

    Charge charge = Charge()
      ..amount = (widget.paymentAmount * 100)
      ..reference = _getReference()
      ..accessCode = await _paymentRepository.createAccessCode(
          _getReference(),
          widget.paymentAmount * 100,
          userDetails.emailAddress.toString(),
          skKey)
      ..email = userDetails.emailAddress.toString();

    CheckoutResponse response = await plugin.checkout(context,
        method:
            CheckoutMethod.selectable, 
        charge: charge,
        fullscreen: true);

    print(response);

    if (response.status == true) {
      var res = await _paymentRepository.verifyOnServer(
          response.reference.toString(), skKey);

      if (res == 'success') {
        var obj = widget.bookingModel;
        obj.paymentReference = response.reference.toString();
        transacRefernceId = response.reference.toString();
        log(jsonEncode(obj));

        ref.read(bookingFutureProvider.notifier).initiateReservation(obj);
      }
    } else {
      if (kDebugMode) {
        print('failed here');
      }
      Navigator.of(context).pop();
    }
  }

**Comment by Leekogram** [comment]
Please can someone answer me Please i running out of time.

**Comment by thekeplertech** [comment]
have you found solution ?

**Comment by Leekogram** [comment]
> have you found solution ?

Nope, what I did was, I just made the app process card payment instead.  And I was thinking of using paystack web plugin in a web view probably if I am still in need of the bank payment
